README.md

```diff
@@ -77,6 +84,12 @@ cargo clippy
 cargo test
 ```
 
+### VS Code support
+
+- Recommended extensions are listed with their extension IDs in `.vscode/extensions.json`.
+
+- If you want VS Code to format your files when you hit save, add `"editor.formatOnSave": true` to your user settings (use ⌘⇧P and search for _Preferences: Open User Settings (JSON)_).
```

Comment:
Isn't this a little too verbose since we're just citing the docs here? I think [a link to the official docs](https://code.visualstudio.com/docs/getstarted/settings) would be enough. It's also biased towards macOS ("use ⌘⇧P and search ..."), no?